GoogleMaps/Fable.Import.GoogleMaps.fsproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <Version>0.1.0</Version>
+    <PackageVersion>0.1.0</PackageVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Fable.Import.GoogleMaps.fs" />
+    <Compile Include="Fable.Helper.GoogleMaps.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Browser\Fable.Import.Browser.fsproj" />
+  </ItemGroup>
+  
```

Comment:
We need to add:

```fs
<ItemGroup>
    <Content Include="*.fsproj; *.fs" PackagePath="fable\" />
</ItemGroup>
```

because you are providing code and not only type definitions in this library. In general, for binding, we don't need the source file because the DLL files are enough and quicker to parse.